js/Story.js

```diff
@@ -224,50 +220,50 @@ Story.prototype.getTemplateForTiddler = function(title,template,tiddler)
 Story.prototype.refreshTiddler = function(title,template,force,customFields,defaultText)
 {
 	var tiddlerElem = this.getTiddler(title);
-	if(tiddlerElem) {
-		if(tiddlerElem.getAttribute("dirty") == "true" && !force)
-			return tiddlerElem;
-		template = this.chooseTemplateForTiddler(title,template);
-		var currTemplate = tiddlerElem.getAttribute("template");
-		if((template != currTemplate) || force) {
-			var tiddler = store.getTiddler(title);
-			if(!tiddler) {
-				tiddler = new Tiddler();
-				if(store.isShadowTiddler(title)) {
-					var tags = [];
-					tiddler.set(title,store.getTiddlerText(title),config.views.wikified.shadowModifier,version.date,tags,version.date);
-				} else {
-					var text = template==config.tiddlerTemplates[DEFAULT_EDIT_TEMPLATE] ? // #166
-								config.views.editor.defaultText.format([title]) :
-								config.views.wikified.defaultText.format([title]);
-					text = defaultText || text;
-					var fields = customFields ? customFields.decodeHashMap() : null;
-					tiddler.set(title,text,config.views.wikified.defaultModifier,version.date,[],version.date,fields);
-				}
-			}
-			tiddlerElem.setAttribute("tags",tiddler.tags.join(" "));
-			tiddlerElem.setAttribute("tiddler",title);
-			tiddlerElem.setAttribute("template",template);
-			tiddlerElem.onmouseover = this.onTiddlerMouseOver;
-			tiddlerElem.onmouseout = this.onTiddlerMouseOut;
-			tiddlerElem.ondblclick = this.onTiddlerDblClick;
-			tiddlerElem[window.event?"onkeydown":"onkeypress"] = this.onTiddlerKeyPress;
-			tiddlerElem.innerHTML = this.getTemplateForTiddler(title,template,tiddler);
-			applyHtmlMacros(tiddlerElem,tiddler);
-			if(store.getTaggedTiddlers(title).length > 0)
-				jQuery(tiddlerElem).addClass("isTag");
-			else
-				jQuery(tiddlerElem).removeClass("isTag");
-			if(store.tiddlerExists(title)) {
-				jQuery(tiddlerElem).removeClass("shadow");
-				jQuery(tiddlerElem).removeClass("missing");
+	if(!tiddlerElem) return;
+	if(tiddlerElem.getAttribute("dirty") == "true" && !force)
```

Comment:
Hello Mario, could you help me with separating different commits into different pull requests?

Actually most of the changes in this pull request originate from commits that I've made just into my fork to the code clean the code from load of excessive brackets and things like the one you're asking about (I explain that one below). I was hoping to be able to push only one commit (and actually I also had to separate the filtering change from the code "formatting" change), but haven't found such option in GitHub and had to push all 5 commits altogether.

In short, what I really wanted to push are lines 46 and 51 in https://github.com/TiddlyWiki/tiddlywiki/pull/194/commits/eef40d027d5f077472e67a8fcab58aed5d4b4428. It's ok if you just change them in the main repo without pulling this.

As for the other commits, those are mostly about readability of the code. I regularly look through the code via broswers' "open source code" viewer or console and `.toString()` method which doesn't highlight syntax and usually feels like it's clogged with unnecessary brackets and indentation. As for the change you're asking here, it has the following structure (and is about extra indentation):
were:

    var tiddlerElem = this.getTiddler(title);
    if(tiddlerElem) {
        // loads of code which gets extra indentation
    }
    return tiddlerElem;

became:

    var tiddlerElem = this.getTiddler(title);
    if(!tiddlerElem) return;
    // loads of code without extra indentation
    return tiddlerElem;

Although most of this wasn't intended for pushing, I can't help asking about

1. what do you think about code styling already mixed in the core? (there's lots of `if` .. `else` without curly brackets where they can be omitted and lots of those where they are preserved) Should all the omitted brackets be "returned"?
2. do you think reducing file size and making code shorter has low importance compared to a possibility to add `console.log` without adding a pair of brackets (which can be done easily) and being less error-prone (taking into account that the development of TWc core almost stopped)?